sonic-xcvrd/setup.py

```diff
@@ -21,10 +21,14 @@
         'enum34; python_version < "3.4"',
         'sonic-py-common',
     ],
-    setup_requires=[
+    setup_requires = [
         'wheel'
     ],
-    classifiers=[
+    tests_require = [
+        'pytest',
+        'pytest-cov',
+    ],
+    classifiers = [
```

Comment:
Remove spaces around `=`, as this is technically a named parameter to the `setup()` function.

Reply:
fixed